codalab/lib/bundle_cli.py

```diff
@@ -1922,11 +1922,11 @@ def do_mimic_command(self, args):
         'macro',
         help=[
             'Use mimicry to simulate macros.',
-            '  macro M A B   <=>   mimic M-in1 M-in2 M-out A B'
+            '  macro M A B named-input:C <=> mimic M-in1 M-in2 M-in-named-input M-out A B C'
```

Comment:
can you make it clearer what the generalization is?
`<name1>:C <name2>:D` ...

Reply:
Done